doc/sphinx/changes.rst

```diff
@@ -465,17 +563,20 @@ Command-line tools
   Makefiles produced by ``coq_makefile`` now use ``.DELETE_ON_ERROR``
   (`#14238 <https://github.com/coq/coq/pull/14238>`_,
   by Gaëtan Gilbert).
+- **Removed:**
+  Previously deprecated command line options
+  ``-sprop-cumulative`` and ``-input-state`` and its alias ``-is``
+  (`#13822 <https://github.com/coq/coq/pull/13822>`_,
+  by Gaëtan Gilbert).
```

Comment:
Why has this been moved up? We generally try to keep changes gathered by categories. Here this inserts a **Removed** between two **Changed**.

Reply:
It's just to group all the native stuff together.

Reply:
Hum, in this case what about inserting a new category? This is something that we've done in the past when there were many related changes (8.12 had "Reals library", "Extraction" and "Reference manual" in addition to the usual stuff). It doesn't have to be a new permanent category.